### PR TITLE
Use mc help svg

### PR DESCRIPTION
**Pull request by jasonLaster**
### summary

this updates the svg logic so we can re-use mc svgs

### CODE REVIEW

**Comment by jasonLaster** [comment]
:thumbsup:

**Comment by darkwing** [comment]
Added a second commit to remove the asset.  I'm still not seeing the icon though.

**Comment by darkwing** [comment]
Whether a comedy of errors on my part or legit weirdness, using `/mc/` created nothing but problems for me; I was thinking the conflict was here:

```js
.replace(/(chrome:\/\/|resource:\/\/)/, "/mc/")
```

But no changes I made fixed the issue.  I'm also a bit dubious about using `mc` in our CSS, as most contributors wont know what it represents; I think a more explicit path would be helpful.

File: src/components/SecondaryPanes/CommandBar.css [hunk 1]
```diff
@@ -92,7 +92,7 @@ img.shortcuts {
 }
 
 .command-bar img.shortcuts {
-  mask: url(/images/help.svg) no-repeat;
+  mask: url(/mozillacentral/help.svg) no-repeat;
```
Comment (jasonLaster) [inline src/components/SecondaryPanes/CommandBar.css]: weird maybe the problem was that images should have been mc

**Comment by jasonLaster** [comment]
Could you check one more time:

**Pull request merged**